### ISSUE TITLE
Analyzers: AdditionalFiles not used

### ISSUE THREAD

**Issue by AntyMew** [OmniSharp/omnisharp-roslyn#1510]
OS: Debian buster
Dev env: vscode 1.34.0 + omnisharp-roslyn 1.19.1 + StyleCop.Analyzers 1.1.118

Given this example code:
```cs
// <copyright file="Program.cs" company="Callie LeFave">
// Copyright (c) Callie LeFave 2019
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at https://mozilla.org/MPL/2.0/.
// </copyright>

using System;

namespace testproj
{
    public class Program
    {
        public static void Main()
        {
            Console.WriteLine("Hello world!");
        }
    }
}
```

And this stylecop.json:
```json
{
    "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
    "settings": {
        "documentationRules": {
            "companyName": "Callie LeFave",
            "copyrightText": "Copyright (c) {companyName} {copyrightYear}\n{licenseNotice}",
            "variables": {
                "copyrightYear": "2019",
                "licenseNotice": "This Source Code Form is subject to the terms of the Mozilla Public\nLicense, v. 2.0. If a copy of the MPL was not distributed with this\nfile, You can obtain one at https://mozilla.org/MPL/2.0/."
            }
        },
        "orderingRules": {
            "usingDirectivesPlacement": "outsideNamespace"
        }
    }
}
```

On build, the following warnings are raised:
```
Program.cs(12,18): warning CS1591: Missing XML comment for publicly visible type or member 'Program' [/home/callie/.local/src/testproj/testproj.csproj]
Program.cs(14,28): warning CS1591: Missing XML comment for publicly visible type or member 'Program.Main()' [/home/callie/.local/src/testproj/testproj.csproj]
Program.cs(12,18): warning SA1600: Elements should be documented [/home/callie/.local/src/testproj/testproj.csproj]
Program.cs(14,28): warning SA1600: Elements should be documented [/home/callie/.local/src/testproj/testproj.csproj]
```

But in the editor, [SA1636](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1636.md) and [SA1200](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1200.md) are also raised, despite the configuration:

![2019-05-30-194436_1920x1080_scrot](https://user-images.githubusercontent.com/16222300/58672136-7ca02c80-8313-11e9-9dfd-b8fe8e0797ea.png)

Thank you for your time!

**Comment by savpek** [comment]
Not sure about this since i am not familiar with the custom configuration files but i think this feature isn't implemented at all at omnisharp. Theres methods like http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/Workspace/Workspace.cs,15c1644018187c11,references in workspace base but they are internal so i am not sure is this possible to implement without reflection? cc @rchande @filipw any insights for this?

**Comment by rchande** [comment]
cc @heejaechang @mavasani

**Comment by filipw** [comment]
fixed by #1547

**Comment by AntyMew** [comment]
Confirmed fixed, thank you!
